source/ed/ed2nav.cpp

```diff
@@ -165,6 +164,41 @@ struct FindAdminWithCities {
     }
 };
 
+bool rename_old_file(const std::string& output_filename, std::string& backup_output_filename) {
+    auto logger = log4cplus::Logger::getInstance("ed2nav::rename_old_file");
+    if (boost::filesystem::exists(output_filename)) {
+        if (rename(output_filename.c_str(), backup_output_filename.c_str()) != 0) {
+            LOG4CPLUS_ERROR(logger, "Unable to rename old data file: " << std::strerror(errno));
+            return false;
+        }
+        LOG4CPLUS_INFO(logger, "Old data file successfully renamed to " << output_filename << ".bak");
+    }
+    return true;
+}
+
+bool rename_temp_file(const std::string& temp_output_filename, const std::string& output_filename) {
+    auto logger = log4cplus::Logger::getInstance("ed2nav::rename_temp_file");
+    if (rename(temp_output_filename.c_str(), output_filename.c_str()) != 0) {
+        LOG4CPLUS_ERROR(logger, "Unable to rename temp data file: " << temp_output_filename << std::strerror(errno));
+        return false;
+    }
+    LOG4CPLUS_INFO(logger, "File: " << output_filename << " saved");
+    return true;
+}
```

Comment:
Minor: Two functions above can be mutualized non?

Reply:
Right, thanks !